climate_data.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import os
+
+def load_city_climate_data(city_folder):
```

Comment:
Write a function to load a single data file. Then another to load them all. If you need to make a change, it's super easy to find and update.